editor.planx.uk/src/@planx/components/Send/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { TYPES } from "@planx/components/types";
+import { Store } from "pages/FlowEditor/lib/store";
+
+export function findGeoJSON(
+  flow: Store.flow,
+  breadcrumbs: Store.breadcrumbs
+): { type: "Feature" } | undefined {
+  const boundaryQuestionID = Object.keys(breadcrumbs).find(
+    (questionId) => flow[questionId]?.type === TYPES.DrawBoundary
```

Comment:
nit: rather than `questionId` we often refer to keys of the breadcrumbs object as `nodeId` (to avoid confusion with "question" also being a specific component type too)